src/test/scala/uk.gov.nationalarchives.fileformat/ExternalServicesTest.scala

```diff
@@ -0,0 +1,72 @@
+package uk.gov.nationalarchives.fileformat
+
+import java.io.File
```

Comment:
`import java.io.File` appears to be unnecessary import

Reply:
It is, I've taken out the line breaks around it as well :smile: